internal/providers/terraform/aws/testdata/waf_web_acl_test/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
this file should not be checked in (I updated the .gitignore in master earlier today)